README.md

```diff
@@ -137,7 +138,9 @@ withFailure
 withFlowDetermination
```

Comment:
I'd update this list, following `with'Sarasa'` injections https://github.com/Wolox/redux-recompose/tree/master/src/injections